components/browser/engine-gecko-nightly/src/test/java/org/mozilla/geckoview/MockRecordingDevice.kt

```diff
@@ -0,0 +1,17 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.geckoview
+
+import mozilla.components.test.ReflectionUtils
+
+class MockRecordingDevice(
```

Comment:
nit: Should this be `internal`?

Reply:
It could. Since it's in `test` it's not super important though: This code is internal anyways. :)